templates/web/base/reports/_list-filters.html

```diff
@@ -27,26 +28,39 @@
   [% END %]
 [% END %]
 
-<div class="report-list-filters-wrapper">
+<div class="report-list-filters-wrapper govuk-fieldset-wrapper govuk-small">
 
 [% IF use_form_wrapper %]
     <form method="get" action="">
 [% END %]
 
-        <p class="report-list-filters">
-          [% tprintf(loc('<label for="statuses">Show</label> %s reports <label for="filter_categories">about</label> %s', "The first %s is a dropdown of all/fixed/etc, the second is a dropdown of categories"), mark_safe(select_status), mark_safe(select_category)) %]
-          <input type="submit" name="filter_update" value="[% loc('Go') %]">
-        </p>
+        <fieldset>
+            <div class="multi-select-filter-group">
+                <div class="report-list-filters">
+                    [% tprintf(loc('<label for="statuses">Report status</label> %s', "Dropdown of all/fixed/etc"), mark_safe(select_status)) %]
+                    <input type="submit" name="filter_update" value="[% loc('Go') %]">
```

Comment:
There are still two go buttons, one under category, one under show older reports; would be good to only have one at the end (so after the show older tickbox, outside the page==around check, I guess?)

Reply:
@dracos I added [this commit ](https://github.com/mysociety/fixmystreet/pull/4728/commits/948c6ce17c8a07f862aa6da41af660279d7396a4), I did a minor tweak to the JS, because the markup couldn't target the button anymore, for being outside of `.report-list-filters`